### ISSUE TITLE
Binding version

### ISSUE THREAD

**Issue by jeapostrophe** [wilbowma/scribble-minted#5]
Hi Will,

I adapted this code and made a version that works HTML and walks the turning it into Scribble and optionally creating xrefs from them. You can see the code here:

https://github.com/reach-sh/reach-lang/blob/master/docs-src/minted/main.rkt

**Comment by wilbowma** [comment]
If I use this, will it generate the "right" output in the PDF backend, since it's converted back to Scribble in the process?

**Comment by jeapostrophe** [comment]
I don't think so, because it uses `xexpr-property`, but I think a strategy like this would work.

**Comment by wilbowma** [comment]
Okay thanks. I'll give it some thought.
